src/app/registration/facility-select/facility-select.component.spec.ts

```diff
@@ -329,7 +330,7 @@ describe('FacilitySelectComponent', () => {
               '6': true,
             },
             bookableHolidays: [],
-            bookingDaysRichText: ''
+            bookingDaysRichText: '',
```

Comment:
Same here

Reply:
Resolved in [Commit 43beab5](https://github.com/bcgov/parks-reso-public/pull/461/commits/43beab52a1c7d818910d777c2133aefa47ce60ff)